lib/analytics/page.rb

```diff
@@ -0,0 +1,72 @@
+module Analytics
+  class Page
```

Comment:
it would be cheap, and easy with redis' sorted set data type:  http://redis.io/commands#sorted_set
```ruby
page_id = 5
redis.zincrby 'pages:all:total_actions',  1, page_id
redis.zincrby 'pages:all:total_new_members',  1, page_id
```

I'll update this PR with some global counters, along with some API endpoints for accessing that data.